python/ray/_private/function_manager.py

```diff
@@ -227,61 +243,55 @@ def get_execution_info(self, job_id, function_descriptor):
         Returns:
             A FunctionExecutionInfo object.
         """
-        if self._worker.load_code_from_local:
-            # Load function from local code.
-            # Currently, we don't support isolating code by jobs,
-            # thus always set job ID to NIL here.
-            job_id = ray.JobID.nil()
+
+        try:
+            # Try to load function from local code first.
             if not function_descriptor.is_actor_method():
-                self._load_function_from_local(job_id, function_descriptor)
-        else:
+                self._load_function_from_local(function_descriptor)
+        except:
             # Load function from GCS.
```

Comment:
Add comment: "If the function doesn't exist in local code, try loading it from GCS."